developer/guides/troubleshooting/layout-shift.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Investigating a layout shift
```

Comment:
What about update this to be the problem the user is encountering? For example, "Layout shift on initial page load". This would be in line with the other troubleshooting guide, "Builder isn't loading".